mkt/websites/models.py

```diff
@@ -156,6 +156,9 @@ class Website(ModelBase):
     # Icon cache-busting hash.
     icon_hash = models.CharField(max_length=8, blank=True)
 
+    # Promo image cache-busting hash.
+    promo_img_hash = models.CharField(max_length=8, blank=True, null=True)
```

Comment:
Do we need a migration to generate the hashes of existing websites? Or are there no existing promo images?